src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -704,6 +706,42 @@ public function exportAction(CustomerFilters $filters)
             ->setFileName('customer_' . date('Y-m-d_His') . '.csv');
     }
 
+    /**
+     * @AdminSecurity("is_granted(['read'], request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function getCustomerCartsAction(Request $request)
```

Comment:
If we get a lot of these controller actions, we might create a dedicated Controller. Maybe something like `CustomerAjaxController`. Or rename existing controller to `CustomerPageController` and have a `UtilsCustomerController`.

Names are not very good 😅but you get the idea I think